From 324712460501a808b706fe72969a68ce37fced3b Mon Sep 17 00:00:00 2001
From: Denys Stakhnik <denys.stakhnik@globallogic.com>
Date: Fri, 14 Aug 2020 16:58:51 +0300
Subject: [PATCH 2/2] [WORKAROUND] force client composition in surfaceflinger

temporary disabled composition by HWC due to a issues
with transparent views/layouts.

Signed-off-by: Denys Stakhnik <denys.stakhnik@globallogic.com>
Change-Id: I0725606bd0aa5f432e55368d16c23a30f340836c
Jira-id: ADM-4920
---
 services/surfaceflinger/CompositionEngine/src/OutputLayer.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/services/surfaceflinger/CompositionEngine/src/OutputLayer.cpp b/services/surfaceflinger/CompositionEngine/src/OutputLayer.cpp
index 98e3a71d4..035e8f568 100644
--- a/services/surfaceflinger/CompositionEngine/src/OutputLayer.cpp
+++ b/services/surfaceflinger/CompositionEngine/src/OutputLayer.cpp
@@ -311,6 +311,8 @@ void OutputLayer::updateCompositionState(
         forceClientComposition) {
         state.forceClientComposition = true;
     }
+
+    state.forceClientComposition = true;
 }
 
 void OutputLayer::writeStateToHWC(bool includeGeometry) {
-- 
2.28.0

